Apeswap-Bots/New-Pair-Creation/src/utils.ts

```diff
@@ -0,0 +1,34 @@
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+import { APEFACTORY_ABI } from "./constants";
+
+const { CREATE_PAIR_FUNCTION } = APEFACTORY_ABI;
+
+// finding type definition
+type newPairFindingType = {
+  tokenAAddress: string;
+  tokenBAddress: string;
+};
+
+type newPairParamsType = {
+  functionSig: string;
+};
```

Comment:
I don't like too much the usage of this wrappers, they add an unnecessary complexity in the code, and the name is not self-descriptive, I don't know what the type contains o what it is meant for by reading the name